### DISCUSSION TITLE
Student indentification veryfied, but the pack is not recieved

### DISCUSSION THREAD

**Discussion by aprylewu** [community/community#77023] [GitHub Education]
![image](https://github.com/community/community/assets/129849514/ffe2a5e6-2f53-4a9a-9b52-aeb172c89601)
![image](https://github.com/community/community/assets/129849514/8c353492-68f4-400e-a06f-876cbf98e680)

even 12hrs after my veryfication, in vscode it still shows me that my access to academic benefits is still not avaliable, is that normal?:
[INFO] [auth] [2023-11-26T11:41:50.858Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-11-26T11:41:50.859Z] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as Rabbitwu2333."

**Comment by rskbansal** [comment]
Hey @Rabbitwu2333 
Might be possible that you haven't enabled Copilot from your GitHub account settings. To do the same, follow the below mentioned steps :

- In the upper-right corner of any page, click your profile photo, then click Settings
- In the "Code, planning, and automation" section of the sidebar, click 'Copilot'
- Click on **Enable GitHub Copilot**
- Now it shouldn't be asking you for the payment details & instead directly take you to preferences
- Click **Save and get started**

You can also visit this URL for the settings page - https://github.com/settings/copilot

Hope this answer helped you, enjoy your GitHub Copilot subscription :)

**Reply by aprylewu** [reply]
I didnt even see a "enable" button, I guess I still have to wait until my pack is avaliable, but just wondering how long this would take......

**Reply by TtuHamg** [reply]
hi, i have the same question. when i use copilot in vscode, i meet the same error:"Invalid copilot token: missing token: 403 " and "Extension activation failed: "Contact Support. You are currently logged in as TtuHamg."" I want to know if your problem has been resolved?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Rabbitwu2333 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by HugoCodeX** [comment]
I am in the same situation, my plan to study is not available.

**Comment by Shebaby55** [comment]
My  application was aslo veryfied On Nov 26,  and I haven't received my pack until now
